unity-client/Assets/Scripts/MainScripts/DCL/Controllers/Scene/Tests/BlockerHandlerShould.cs

```diff
@@ -5,42 +5,167 @@
 using UnityEngine;
 using UnityEngine.Assertions;
 using UnityEngine.TestTools;
+using DCL;
+using System.Linq;
 
 namespace Tests
 {
     public class BlockerHandlerShould : TestsBase
     {
-        protected override bool justSceneSetUp => true;
+        protected override bool enableSceneIntegrityChecker => false;
+
+        const string PARCEL_BLOCKER_POOL_NAME = "ParcelBlocker";
+        WorldBlockersController worldBlockersController;
+        BlockerHandler blockersHandler;
+        Dictionary<Vector2Int, PoolableObject> blockers;
+
+        [UnitySetUp]
+        protected override IEnumerator SetUp()
+        {
+            yield return base.SetUp();
+
+            worldBlockersController = Reflection_GetField<WorldBlockersController>(sceneController, "worldBlockersController");
+
+            Assert.IsNotNull(worldBlockersController);
+
+            blockersHandler = Reflection_GetField<BlockerHandler>(worldBlockersController, "blockerHandler");
+            blockers = Reflection_GetField<Dictionary<Vector2Int, PoolableObject>>(blockersHandler, "blockers");
+
+            if (!PoolManager.i.ContainsPool(PARCEL_BLOCKER_POOL_NAME))
+            {
+                GameObject go = Object.Instantiate(Reflection_GetStaticField<GameObject>(typeof(BlockerHandler), "blockerPrefab"));
+                Pool pool = PoolManager.i.AddPool(PARCEL_BLOCKER_POOL_NAME, go);
+                pool.persistent = true;
+                pool.ForcePrewarm();
+            }
```

Comment:
The tests of this suite are setting up everything using the SDK from the ground up when we should just try to test only the `BlockerHandler` and `BlockerHandlerController` classes. I think this is very related to the stuff we were discussing today. If we do this correctly I think we can manage to write tests that are a LOT shorter, and less vulnerable to CI false negatives.